Create ui-c8y-1023-25-5-enhanced-translation-options-for-widget-and-dashboard-titles-with-help-popover.md (#4193)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-25-5-enhanced-translation-options-for-widget-and-dashboard-titles-with-help-popover.md

---
date: ""

> title: Enhanced translation options for widget and dashboard titles with help popover (#10872) [GRAFT][release/cd] (#11051)

Reviewer: ```suggestion
title: Clearer translation options for dashboard and widget titles
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64618
version: 1023.25.5
---

> Improved clarity of translation options for dashboard and widget titles, added popover with additional info.

Reviewer: ```suggestion
When configuring dashboards and widgets, you can choose to translate their titles if possible. Previously, these options were not explained enough. Now, they have been placed under common label and a help popover has been added that provides additional info and guidance on how to use this translation feature.
```

> When configuring dashboards and widgets, you can choose to translate their titles if possible. Previously, these options were not explained enough. Now, they have been placed under common label and a help popover has been added that provides additional info and guidance on how to use this translation feature.

Reviewer 2: ```suggestion
When configuring dashboards and widgets, you can choose to translate their titles if possible. Previously, the translation options were not explained clearly enough. Now, they have been placed under a common label, and a help popover has been added that provides additional information and guidance on how to use these translation options.
```
